app/views/comments/_comment_form.html.erb

```diff
@@ -103,7 +103,7 @@
       <p class="submit actions">
         <%= f.submit button_name, :id => "comment_submit_for_#{commentable.id}", data: {disable_with: ts("Please wait...")} %>
         <% if controller.controller_name == 'inbox' %>
-          <a name="comment_cancel" id="comment_cancel"><%= ts("Close") %></a>
+          <a name="comment_cancel" id="comment_cancel" style="cursor: default"><%= ts("Cancel") %></a>
```

Comment:
We generally ask that pull requests not stray beyond the parameters of the Jira issue unless it's unavoidable. However, because the change required for the issue is so small, we can make a one-time exception as long as you provide testing instructions.

That being said, we don't use inline CSS in our code -- any style changes should be made in the stylesheets. 

It looks like this is the source of the move cursor: 

https://github.com/otwcode/otwarchive/blob/2e1b6f578e6545b723f09d6d601d35b533070d94/public/stylesheets/site/2.0/07-interactions.css#L288-L291

So maybe we should be setting the cursor for `.ui-draggable a` to default?

Reply:
OK got it - commit amended.